Libraries/Cordio/platform/targets/maxim/max32655/sources/pal_cfg.c

```diff
@@ -22,6 +22,10 @@
 #include "ll_defs.h"
 #include "mxc_sys.h"
 
+#ifndef PAL_CFG_LL_MAX
```

Comment:
This will minimize the SRAM utilization by default, option to include more features.